src/Footer.js

```diff
@@ -1,9 +1,33 @@
 import React, { Component } from 'react'
+import Navbar from 'react-bootstrap/Navbar'
+import Container from 'react-bootstrap/Container'
 
 class Footer extends Component {
+
+  constructor(props){
+  super(props);
+  this.state = {
+      likes: 0, 
+      selected: false
+  }
+}
+
+handleClick = () => {
+  this.setState({
+      likes: this.state.likes + 1
+  });
+      this.setState({
+          selected: !this.state.selected
+      });

```

Comment:
Fun tip, since state is an object, you can just add `selected: !this.state.selected` inside the first setState. 👍